From f4e4af1c01fd51ccf7f368a7f2e1157f0d73363c Mon Sep 17 00:00:00 2001
From: LAK132 <1386467+LAK132@users.noreply.github.com>
Date: Sun, 6 Aug 2023 13:41:22 +0930
Subject: [PATCH] Fix path to imgui headers

---
 ImFileDialog.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ImFileDialog.cpp b/ImFileDialog.cpp
index 53832d9..775b459 100644
--- a/ImFileDialog.cpp
+++ b/ImFileDialog.cpp
@@ -7,8 +7,8 @@
 #include <algorithm>
 #include <sys/stat.h>
 #define IMGUI_DEFINE_MATH_OPERATORS
-#include <imgui/imgui.h>
-#include <imgui/imgui_internal.h>
+#include <imgui.h>
+#include <imgui_internal.h>
 
 #define STB_IMAGE_IMPLEMENTATION
 #include "stb_image.h"
-- 
2.37.1 (Apple Git-137.1)

