From 118e66f95f4ed212b338868722b3195e8a19c723 Mon Sep 17 00:00:00 2001
From: LAK132 <1386467+LAK132@users.noreply.github.com>
Date: Sun, 6 Aug 2023 14:41:47 +0930
Subject: [PATCH] Don't implement stb_image inside ImFileDialog

---
 ImFileDialog.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ImFileDialog.cpp b/ImFileDialog.cpp
index 775b459..8adb1d8 100644
--- a/ImFileDialog.cpp
+++ b/ImFileDialog.cpp
@@ -10,7 +10,6 @@
 #include <imgui.h>
 #include <imgui_internal.h>
 
-#define STB_IMAGE_IMPLEMENTATION
 #include "stb_image.h"
 
 #ifdef _WIN32
-- 
2.37.1 (Apple Git-137.1)

